web/components/feed/feed-items.tsx

```diff
@@ -298,35 +292,46 @@ export function CommentInput(props: {
                 </>
               </>
             )}
-            <div className="mt-2">
-              <Textarea
-                value={comment}
-                onChange={(e) => setComment(e.target.value)}
-                className="textarea textarea-bordered w-full resize-none"
-                placeholder="Add a comment..."
-                rows={answerOutcome == undefined || focused ? 3 : 1}
-                onFocus={() => setFocusedIfFreeResponse(true)}
-                onBlur={() => !comment && setFocusedIfFreeResponse(false)}
-                maxLength={MAX_COMMENT_LENGTH}
-                onKeyDown={(e) => {
-                  if (e.key === 'Enter' && (e.ctrlKey || e.metaKey)) {
-                    submitComment(id)
-                  }
-                }}
-              />
-              <button
-                className={
-                  'btn btn-outline btn-sm text-transform: mt-1 capitalize'
-                }
-                onClick={() => {
-                  submitComment(id)
-                  setFocusedIfFreeResponse(false)
-                }}
-              >
-                {user ? 'Comment' : 'Sign in to comment'}
-              </button>
-            </div>
+            {(answerOutcome === undefined || focused) && (
+              <div className="mt-2">
+                <Textarea
+                  value={comment}
+                  onChange={(e) => setComment(e.target.value)}
+                  className="textarea textarea-bordered w-full resize-none"
+                  placeholder="Add a comment..."
+                  autoFocus={true}
+                  rows={answerOutcome == undefined || focused ? 3 : 1}
+                  onFocus={() => setFocused(true)}
+                  onBlur={() => !comment && setFocused(false)}
```

Comment:
As discussed, might want to make this simply onBlur={() => setFocused(false)} and then add above an OR condition to render the text area if `comment` has text.